applications/vanilla/controllers/class.discussioncontroller.php

```diff
@@ -44,7 +44,7 @@ public function __get($name) {
                 return $this->data('Comments');
                 break;
         }
-        throw new Exception("DiscussionController->$name not found.", 400);
+        return null;
```

Comment:
Is this supposed to be part of the PR?

Reply:
it is unrelated, but intentional.

Reply:
This was previously a `Gdn_Controller`, not an API controller, and checking for the existence of a properly on `DiscussionController` would cause an exception to be thrown. Pretty ridiculous.